modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_fields.rb

```diff
@@ -4,6 +4,10 @@ module AppealsApi
   module PdfConstruction
     module HigherLevelReview::V2
       class FormFields
+        def middle_initial
```

Comment:
This field is not auto filled and needs to be printed in the pdf form field, which had a name change with the new version.